account_fiscal_position_vies_warning/models/res_partner.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2025 Moduon Team S.L. <info@moduon.team>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, models
+from odoo.exceptions import ValidationError
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    @api.constrains("property_account_position_id")
+    def check_fiscal_position_and_vies_valid(self):
+        if not self.env.company.vat_check_vies:  # Verify VAT Numbers is True
```

Comment:
Code is self explanatory in this case
```suggestion
        if not self.env.company.vat_check_vies:
```